std_msgs/Header header
zed_obj_dec_pos_tag/Object[] object_array 